lib/Partitioner/Partitioner.cpp

```diff
@@ -1209,7 +1209,7 @@ llvm::Error Partitioner::QuantizationProfilingPartition(
   FunctionToBackendNameMap funcToBackend;
   funcToBackend = backendBasedPartition(F_, backends, cctx);
   module_->eraseFunction(F_);
-  auto backend = createBackend(profilingBackend);
+  std::unique_ptr<Backend> backend(createBackend(profilingBackend));
```

Comment:
did we exercise this pass in tests? (it's very likely to be reported by asan)

Reply:
So I found it as a result of ASAN complaining when I went to land the BackendTest ported to executionEngine2, not sure how it slipped by before.

Reply:
For quantization in Partitioner, the tests are added in ./tests/images/run.sh.  It is should be in CI, right?

Reply:
It is run in CI but not in ASAN.